modules/backend/models/brandsetting/fields.yaml

```diff
@@ -39,6 +39,15 @@ tabs:
             span: right
             fileTypes: jpg,jpeg,bmp,png,webp,gif,svg,ico
 
+        fallback_image:
+            label: backend::lang.branding.fallback_image
+            type: fileupload
+            commentAbove: backend::lang.branding.fallback_image_description
+            mode: image
+            tab: backend::lang.branding.brand
+            span: right
+            fileTypes: jpg,jpeg,bmp,png,webp,gif,svg,ico
```

Comment:
ico is not a supported general image type in browsers AFAIK. Even bmp would be pushing it a bit.

Reply:
Alright, I can remove them both.

Reply:
As per [the MDN web docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#Supported_image_formats), all these formats mentioned are supported by "Chrome, Edge, Firefox, Internet Explorer, Opera, Safari", the only exception being webp which is not supported by Internet Explorer and Safari. But since you don't want them, I can remove them off thro' a commit.

Reply:
.ico should only be used for favicons, it's not a great format for general images.

Reply:
`.bmp` files should be dropped as well, I can't think of any uses for it.